Vagrantfile

```diff
@@ -70,6 +77,9 @@ Vagrant.configure("2") do |config|
     end
   end
 
+  config.vm.synced_folder ".", "/fossology",
```

Comment:
You should deactivate the sync to `/vagrant` with:
`config.vm.synced_folder ".", "/vagrant", disabled: true`

Reply:
That is annoying, but I think it is still worth it, to call the folder `"/fossology"`